libs/ui/src/theme/typings.ts

```diff
@@ -57,10 +59,15 @@ export interface ThemeUtils {
   flexRowNoWrap: StyledFlattenSimpleInterpolation
 }
 
-declare module 'styled-components' {
-  export interface CowProtocolTheme extends Colors, ThemeUtils {
-    mode: CowSwapTheme
-  }
+export interface WidgetThemeMode {
```

Comment:
This interface properties are not in use, and this interface should not be a part of `CowProtocolTheme` since this interface is generic for all apps and widget is only CoW Swap specific.

Reply:
Addressed now